addons/website_slides/views/website_slides_templates.xml

```diff
@@ -153,7 +153,7 @@
             <div class="row" t-if="not search">
                 <!-- Navigation -->
                 <div class="col-lg-8 col-md-12 col-12">
-                    <ul class="nav nav-tabs nav-tabs-border" role="tablist">
+                    <ul class="nav nav-tabs nav-tabs-border o_wslides_channel_type_nav_tabs" role="tablist">
```

Comment:
Should not be necessary I think with shiny '%ss' mechanism.

Reply:
Praise the '%ss'